### ISSUE TITLE
Template for #7383: Change a `_program-areas` File From `-card.yml` to `.md` for `[Insert the current name of the file].yml`

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#7404]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
In the `_program-areas` directory, we want to change a file's name and its extension from `-card.yml` to `.md` so that in a future issue
- this file can be used to generate a webpage for that program area automatically
- and the webpage will have the desired URL.

### Details
- On this team, we use collections in Jekyll to generate some webpages using Markdown files in a collection and an HTML layout. The webpages' URL is based on the name of the Markdown file.
  - For example, a webpage generated from `citizen-engagement.md` will have a URL of https://www.hackforla.org/program-areas/citizen-engagement.

### Action Items
- [ ] In your IDE, find `_program-areas/[Insert the current name of the file].yml`
- [ ] Change the name and extension of the file from
`[Insert the current name of the file].yml`
to
`[Insert the name of the file without -card].md`

### For Merge Team
- [ ] Once the pull request for this issue is merged, go to #7424 and do the following:
  - [ ] Check off this issue under the _Child Issues_ section.
  - [ ] If all the issues in the _Child Issues_ section are checked off, 
    - [ ] uncheck the **Dependency label**
      - <details>
          <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
          <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
        </details>
    - [ ] and close the epic.

### Resources/Instructions
- Info on [Collections in Jekyll](https://jekyllrb.com/docs/collections/)
- [`_program-areas` directory on HfLA's Website repo](https://github.com/hackforla/website/tree/gh-pages/_program-areas)
- This issue is tracked in https://github.com/hackforla/website/issues/7424.

**Comment by HackforLABot** [comment]
Hi @JessicaLucindaCheng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by JessicaLucindaCheng** [comment]
Template writing availability
- Availability: 4 hours
- ETA: Monday, Sepetember 9, 2024

**Comment by JessicaLucindaCheng** [comment]
This template is ready to be reviewed.

**Comment by JessicaLucindaCheng** [comment]
# Tasks
- [ ] Check the answer to the question about the Dependency label usage: ______________
- [ ] Determine if these Action Items need to be updated: 
  >     - [ ] uncheck the **Dependency label**
  >       - <details>
  >           <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
  >           <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
  >         </details>

**Comment by JessicaLucindaCheng** [comment]
# Template Writing Update Report
- **Progress:** Almost done. Just need blocker answered.
- **Blockers:** Need question answered in https://github.com/hackforla/website/issues/7383#issuecomment-2339064355
- **Availability:** TBD; depends on blocker being answered
- **ETA:** TBD; depends on blocker being answered

**Comment by JessicaLucindaCheng** [comment]
- This issue has been fixed in https://github.com/hackforla/website/issues/7425 and is a duplicate.
